rdk/rdk.py

```diff
@@ -558,7 +556,7 @@ def get_deployment_parser(ForceArgument=False, Command="deploy"):
         "--generated-lambda-layer",
         required=False,
         action="store_true",
-        help="[optional] Forces rdk deploy to use the Python(3.6-lib,3.7-lib,3.8-lib,) lambda layer generated by rdk init --generate-lambda-layer",
+        help="[optional] Forces rdk deploy to use the Python(3.7-lib,3.8-lib,) lambda layer generated by rdk init --generate-lambda-layer",
```

Comment:
Not even sure why this parenthetical is here, I doubt it helps much.

```suggestion
        help="[optional] Forces rdk deploy to use the Python(3.7-lib,3.8-lib,3.9-lib) lambda layer generated by rdk init --generate-lambda-layer",
```

Reply:
Resolved in https://github.com/mbeacom/aws-config-rdk/compare/remove-python36-361...mbeacom:aws-config-rdk:remove-python-copy-364?expand=1 -- will rebase once merged and raise PR.